source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailView.tsx

```diff
@@ -250,19 +250,27 @@ export const CompensationRequisitionDetailView: React.FunctionComponent<
           </FlexDiv>
         }
       >
-        <SectionField label="Status" labelWidth="4" valueTestId="compensation-status">
+        <SectionField label="Status" labelWidth={{ xl: '4' }} valueTestId="compensation-status">
```

Comment:
I went with xl here instead of xs as these fields are displaying on a very small portion of the screen, so displaying the value after the label reduces the risk of overlap.